test/src/snapshot/mod.rs

```diff
@@ -199,5 +199,17 @@ fn clean_output(output: &str) -> String {
         .unwrap();
     let result = r.replace(&result, "$1???");
 
+    // Remove individual test duration time and gas cost
+    let r = Regex::new("(test .+ )(\\(.*? gas\\))").unwrap();
```

Comment:
Why removing gas usage? I think it is interesting to see if a change increase or decrease gas usage across the board.

Reply:
Agree and done.

Although, gas cost is always zero in tests when I run them no matter which tests. I just thought displaying gas is still not implemented but now when you say I must ask myself am I doing something wrong :thinking: I am talking about simply running `forc test`. The gas is always zero.

Reply:
I noticed this as well. @kayagokalp did some digging and tracked down this PR https://github.com/FuelLabs/sway/pull/6731 was when tests no longer reported any gas usage. @xunilrj do you have any idea why that PR may have caused this?

Reply:
No. I will take a look.